build is failing
```
======================================================================
ERROR: testReportXFormTransform (corehq.apps.reports.tests.test_pillows_cases.testReportCaseProcessing)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_pillows_cases.py", line 509, in testReportXFormTransform
    processed_form = report_pillow.change_transform(form)
  File "/home/travis/build/dimagi/commcare-hq/corehq/pillows/reportxform.py", line 27, in change_transform
    doc_ret = super(ReportXFormPillow, self).change_transform(doc_dict, include_props=False)
  File "/home/travis/build/dimagi/commcare-hq/corehq/pillows/xform.py", line 74, in change_transform
    case_blocks = extract_case_blocks(doc_ret)
  File "corehq/ex-submodules/casexml/apps/case/xform.py", line 454, in extract_case_blocks
    return [struct if include_path else struct.caseblock for struct in _extract_case_blocks(doc)]
  File "corehq/ex-submodules/casexml/apps/case/xform.py", line 484, in _extract_case_blocks
    for case_block in _extract_case_blocks(value, path=new_path):
  File "corehq/ex-submodules/casexml/apps/case/xform.py", line 481, in _extract_case_blocks
    case_block.get('@date_modified'), none_ok=True)
  File "corehq/ex-submodules/couchforms/validators.py", line 11, in validate_phone_datetime
    raise PhoneDateValueError('{!r}'.format(datetime_string))
PhoneDateValueError: ''
```

--- a/corehq/ex-submodules/couchforms/validators.py
+++ b/corehq/ex-submodules/couchforms/validators.py
@@ -1,10 +1,17 @@
 from iso8601 import iso8601
+from corehq.util.soft_assert import soft_assert
 from couchforms.exceptions import PhoneDateValueError
+
+_soft_assert = soft_assert('@'.join(['droberts', 'dimagi.com']))
 
 
 def validate_phone_datetime(datetime_string, none_ok=False):
-    if datetime_string is None and none_ok:
-        return None
+    if none_ok:
+        if datetime_string is None:
+            return None
+        if not _soft_assert(datetime_string != '',
+                            'phone datetime should never be empty'):
+            return None
     try:
         return iso8601.parse_date(datetime_string)
     except iso8601.ParseError:

